src/useGet.ts

```diff
@@ -61,32 +61,31 @@ export default function get(options: UseGetOptions): UseGetData {
     return isRef(params) ? params.value : params
   }
 
-  const servicePath = computed<string>(() => model.servicePath)
   const state = reactive<UseGetState>({
+    isPending: false,
+    hasBeenRequested: false,
+    hasLoaded: false,
+    error: null,
+    isLocal: local
+  })
+
+  const computes = {
     item: computed(() => {
       const getterId = isRef(id) ? id.value : id
       const getterParams = isRef(params)
         ? Object.assign({}, params.value)
         : params == null
         ? params
         : { ...params }
-      if (getterParams != null) {
-        return model.getFromStore(getterId, getterParams) || null
-      } else {
-        return model.getFromStore(getterId) || null
-      }
-      // return model.store.state[model.servicePath].keyedById[id.value] || null
-      // return model.getFromStore(id.value) || null
+      return model.getFromStore<M>(getterId, getterParams) || null
     }),
-    isPending: false,
-    hasBeenRequested: false,
-    hasLoaded: false,
-    error: null,
-    isLocal: local
-  })
+    servicePath: computed(() => model.servicePath)
+  }
+
+
 
-  function get<T>(id, params?: Params): T {
-    const idToUse = isRef(id) ? id.value : id
+  function get(id: Id, params?: Params): Promise<M | undefined> {
```

Comment:
One of the tests in `test/use/get.test.ts` called `get()` with no params. Should `id` be an optional parameter and `idToUse` fall back to the `id` passed in to `useGet()`?

Reply:
Resolving for now. This is outside the scope of this PR.